ssv/types.go

```diff
@@ -35,10 +36,12 @@ type AttesterCalls interface {
 
 // ProposerCalls interface has all block proposer duty specific calls
 type ProposerCalls interface {
-	// GetBeaconBlock returns beacon block by the given slot and committee index
-	GetBeaconBlock(slot phase0.Slot, committeeIndex phase0.CommitteeIndex, graffiti, randao []byte) (ssz.Marshaler, spec.DataVersion, error)
-	// GetBlindedBeaconBlock returns blinded beacon block by the given slot and committee index
-	GetBlindedBeaconBlock(slot phase0.Slot, committeeIndex phase0.CommitteeIndex, graffiti, randao []byte) (ssz.Marshaler, spec.DataVersion, error)
+	// SubmitValidatorRegistration submits a validator registration
+	SubmitValidatorRegistration(pubkey []byte, feeRecipient bellatrix.ExecutionAddress, sig phase0.BLSSignature) error
```

Comment:
Because registering validators is needed for proposals using MEV. But since we have a separate duty for validator registrations, I agree that we need to extract this

Reply:
Fixed